generators/generator-base-private.js

```diff
@@ -594,7 +594,7 @@ module.exports = class extends Generator {
         }
         const done = this.async();
         const localModule = path.join(process.cwd(), 'node_modules', blueprint);
-        if (!fs.existsSync(localModule)) {
+        if (fs.existsSync(localModule)) {
```

Comment:
not sure about this. I need to test it. The idea was that if there is no module in your local node_modules folder then we check if there is a global one available. I guess the fix should be to call `done` in the else for this if

Reply:
Yup agree that completely makes sense.